### ISSUE TITLE
Dosen't work in Brazil

### ISSUE THREAD

**Issue by CavalcanteLeo** [frogg/Steal-Phone-Number#1]
Dunno if it is because i have a brazilian number +55 or the bug has been fixed

**Comment by CavalcanteLeo** [comment]
Can't find the classes "ng-binding ng-scope", so, it's not entering in the loop

**Comment by frogg** [comment]
Right now this is only implemented for my mobile provider "Deutsche Telekom" (in Germany).

Does your provider offer a page to check your mobile data usage (for me that's pass.telekom.de)?

**Comment by CavalcanteLeo** [comment]
i have no ideal, but i will check it out, maybe they have

**Comment by fthdgn** [comment]
Some operators also includes phone number on  HTTP headers. I found a website which claim to detect this headers. It seems like my operator does not share my phone number.
http://www.mulliner.org/pc.cgi

**Comment by frogg** [comment]
Good call @fthdgn!
I also found this article online relating to that: https://www.pcworld.com/article/192570/article.html

It's almost 8 years old so I hope providers have already fixed this…but you never know.

**Comment by cherishloveyou** [comment]
absolute,this is not iPhone's bug. It is the Telekom's web bug

**Comment by frogg** [comment]
@cherishloveyou yep, [that's](https://twitter.com/frederikRiedel/status/970631135251369985) what I've always been telling people. This is not related to iOS or WebKit at all, it's a privacy issue with Telekom.
Seboslaw had a great explanation for that on Twitter: https://twitter.com/seboslaw/status/970769247986216960
